browser/ui/views/infobars/web_discovery_infobar_view.cc

```diff
@@ -0,0 +1,51 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/views/infobars/web_discovery_infobar_view.h"
+
+#include <utility>
+
+#include "brave/browser/ui/views/infobars/web_discovery_infobar_content_view.h"
+#include "brave/browser/web_discovery/web_discovery_infobar_delegate.h"
+#include "chrome/browser/ui/views/frame/browser_view.h"
+#include "ui/base/metadata/metadata_impl_macros.h"
+#include "ui/views/widget/widget.h"
+
+std::unique_ptr<infobars::InfoBar> CreateWebDiscoveryInfoBar(
```

Comment:
It'd be good to have a comment where the decl is.

Reply:
Added.